[OV JS]: Add BigInt64/Uint64 support to Tensor and InferRequest

Sujanian1304 opened the pull request:
### Details:
 this fixes issue #32106 

 Added support BigInt64Array and BigUint64Array 
 - Add BigInt64Array and BigUint64Array to the [SupportedTypedArray](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/node/lib/addon.ts#L1).
 - Add tests for using the new types in Tensor and InferRequest.infer()
 
Ticket: CVS-168014

Sujanian1304 commented:
hi @almilosz @Retribution98 ,I have submitted this pull request to Add BigInt64/Uint64 support to Tensor and InferRequest .please let me know your feedback on my PR , Thanks.

Best Regards
Piyush

Retribution98 commented on src/bindings/js/node/tests/unit/infer_request.test.js:
```diff
@@ -131,6 +132,116 @@ describe("ov.InferRequest tests", () => {
+      });
+    });
+
+    it("errors on wrong BigInt64Array size", () => {
```
Please remove this test.

Retribution98 commented on src/bindings/js/node/tests/unit/tensor.test.js:
```diff
@@ -252,6 +252,58 @@ describe("ov.Tensor tests", () => {
       assert.strictEqual(tensor.getSize(), expectedSize);
     });
   });
+  describe("BigInt Tensor support", () => {
```
I double-checked, and we've already tested with big int arrays for Tensor.
Please remove this block.

Sujanian1304 commented:
hi @Retribution98 ,thanks for the feedback.I have addressed all of your suggestions and removed the marked reduntant tests.Please let me know if any other changes are needed.

Retribution98 reviewed (approved):
LGTM

Retribution98 commented:
build_jenkins

Sujanian1304 commented:
hey @Retribution98 @almilosz  ,thanks for approving the PR.Looking forward to contribute more and become a part of the community.
PS feel free to assign issues ,i will be happy to work on them.

Sujanian1304 commented:
Hello! @Retribution98 @almilosz @mlukasze 
Thank you so much for your support and for maintaining this project. I’ve made two contributions so far and plan to focus on applying for GSoC 2026, hopefully with this organization. It would be really helpful if you could suggest which issues I should look at next, recommend any community channels to join, and share how I can communicate best with maintainers and other contributors.
Thanks again—I’m looking forward to learning more!
Piyush kumar
